src/reanimated2/layoutReanimation/defaultTransitions/EntryExitTransition.ts

```diff
@@ -182,7 +182,6 @@ export class EntryExitTransition
           } else {
             return { [transformProp]: 1 } as unknown as TransformProperty;
           }
-          return value;
```

Comment:
But why? :o

Reply:
Because of `'no-unreachable'` There is already a return statement inside if-else condition above.